src/main/java/edu/unc/lib/boxc/migration/cdm/services/PermissionsService.java

```diff
@@ -64,6 +72,54 @@ public void generateDefaultPermissions(PermissionMappingOptions options) throws
                         everyoneField,
                         authenticatedField);
             }
+
+            // works and files
+            if (options.isWithWorks() && options.isWithFiles()) {
+                String workAndFileQuery = "select distinct " + CdmFieldInfo.CDM_ID +
+                        " from " + CdmIndexService.TB_NAME
+                        + " where " + ENTRY_TYPE_FIELD + " = '" + CdmIndexService.ENTRY_TYPE_GROUPED_WORK + "'"
```

Comment:
For consistency, it'd make sense to always address the constants from other classes either with or without the class name, rather than a mix of both styles.